tests/conftest.py

```diff
@@ -265,22 +265,29 @@ def small_array_filters(small_array: np.ndarray, load_scan: LoadScan, filter_con
 
 # IO fixtures
 @pytest.fixture
-def load_scan(loading_config: dict) -> LoadScan:
-    """Instantiate a LoadScan object."""
-    scan_loader = LoadScan(RESOURCES / "minicircle.spm", **loading_config)
+def load_scan(loading_config: dict) -> LoadScans:
+    """Instantiate a LoadScans object."""
+    scan_loader = LoadScans([RESOURCES / "minicircle.spm"], **loading_config)
     return scan_loader
 
 
 @pytest.fixture
-def load_scan_data(load_scan: LoadScan) -> LoadScan:
-    """Instantiate a LoadScan object."""
+def load_scan_data(load_scan: LoadScans) -> LoadScans:
+    """Instantiate a LoadScans object."""
     load_scan.get_data()
     return load_scan
 
 
+#@pytest.fixture
```

Comment:
Yes, the feature and tests are implemented but due to the issue raised by @SylviaWhittle with installing the libasd library, they have been commented out so that a user who knows that they are doing can uncomment and use the feature (or until we hear back from libasd),  but the file loading has been changed.